OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ApplicationController.cs

```diff
@@ -389,19 +389,7 @@ public async Task<IActionResult> Delete(Guid id)
                 return BadRequest(ex.Message);
             }
         }
-
-        private static IEnumerable<ApplicationDto> CreateMultiple(ApplicationApiModel applicationApiModel)
-        {
-            var applications = applicationApiModel.Children.Select(child => new ApplicationDto
-            {
-                ChildId = child.Id,
-                CreationTime = DateTimeOffset.UtcNow,
-                WorkshopId = applicationApiModel.WorkshopId,
-            });
-
-            return applications.ToList();
-        }
-
+        
```

Comment:
maybe remove not only the method but every other objects that were used like `ApplicationApiModel`? Unless it is used somewhere else